<?xml version="1.0" ?><root>
<BetterExceptions><BEversion>v3.11</BEversion><WasBlank>False</WasBlank><TuningLoadFinished>True</TuningLoadFinished><BadObjectCC>False</BadObjectCC><Advice>Not available. More info may be in BE Report.</Advice></BetterExceptions><report><version>2</version><sessionid>bcd7c3737a19ac5e65849f01</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2023-12-21 21:27:50</createtime><buildsignature>Local.Unknown.Unknown.1.103.315.1020-1.300.000.175.Release</buildsignature><categoryid>trait_tracker.py:1738</categoryid><desyncid>bcd7c3737a19ac5e65849f01</desyncid><systemconfig/><screenshot/><desyncdata>[BE Interceptor] BE intercepted this error from a separate mod log so that it can scan for the issue. (AttributeError: 'NoneType' object has no attribute 'min_lod_value')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 179, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 222, in c_api_server_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\sims\masked\v177b\injector.py", line 26, in _wrapped_function&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\sims\masked\v177b\zone_tick.py", line 83, in _turbolib_zone_game_update&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\zone.py", line 507, in update&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\broadcasters\broadcaster_service.py", line 560, in update&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\broadcasters\broadcaster_service.py", line 634, in _update&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\broadcasters\broadcaster.py", line 516, in apply_broadcaster_effect&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\broadcasters\broadcaster_effect.py", line 94, in apply_broadcaster_effect&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\broadcasters\broadcaster_effect.py", line 234, in _apply_broadcaster_effect&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\sims\sim_info_mixin.py", line 553, in add_buff&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\objects\components\__init__.py", line 470, in exported_func&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\objects\components\buff_component.py", line 344, in add_buff&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\buffs\buff.py", line 1215, in on_add&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\buffs\buff.py", line 1263, in apply_all_loot_actions&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\utils\loot.py", line 595, in apply_to_resolver&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\utils\loot_basic_op.py", line 303, in apply_to_resolver&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\utils\loot_ops.py", line 389, in _apply_to_subject_and_target&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\sims\occult\sim_info_with_occult_tracker.py", line 74, in add_trait&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\traits\trait_tracker.py", line 84, in add_trait&#13;&#10;  File "C:\Sims 4 Python Script Workspace (3.7)\My Script Mods\LifeMilestones\Scripts\Lumpinou_LifeMilestones_Injector\Lumpinou_Injector.py", line 6, in _inject&#13;&#10;  File "C:\Sims 4 Python Script Workspace (3.7)\My Script Mods\LifeMilestones\Scripts\Lumpinou_LifeMilsetones_Injections.py", line 372, in lumpinou_milsestones_ckeck_on_game_adding_trait&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\traits\trait_tracker.py", line 654, in _add_trait&#13;&#10;  File "E:\Builds\MCCC_2023_7_1\mc_cmd_center\mc_utils.py", line 202, in _wrapper_function&#13;&#10;  File "E:\Builds\MCCC_2023_7_1\mc_cas\mc_cas_inject.py", line 180, in inject_can_add_trait&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\traits\trait_tracker.py", line 426, in can_add_trait&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\traits\trait_tracker.py", line 1738, in _has_valid_lod&#13;&#10;AttributeError: 'NoneType' object has no attribute 'min_lod_value'rtim=0&#13;&#10;Client Session Time: 16.43&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0x96d01090&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 13,86,0,5599&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19043 GPU: AMD Custom GPU 0405 (RADV VANGOGH) aticfx64.dll 32767.65535.65535.65535, GUID: 00000000-0004-0000-0000-000000000000&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>bcd7c3737a19ac5e65849f01</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2023-12-21 21:49:36</createtime><buildsignature>Local.Unknown.Unknown.1.103.315.1020-1.300.000.175.Release</buildsignature><categoryid>flauschtrud_neighborhoodstories_marriage.py:158</categoryid><desyncid>bcd7c3737a19ac5e65849f01</desyncid><systemconfig/><screenshot/><desyncdata>Exception while updating story progression service.:  (TypeError: object of type 'NoneType' has no len())&#13;&#10;Traceback (most recent call last):&#13;&#10;Caught and logged:&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\story_progression\story_progression_chapter.py", line 382, in &lt;genexpr&gt;&#13;&#10;  File "J:\flauschtrud\Documents\USB Backup\Python Workspace\My Script Mods\NeighborhoodStoriesMarriage\Scripts\flauschtrud_neighborhoodstories_marriage.py", line 158, in reserved_household_slots&#13;&#10;TypeError: object of type 'NoneType' has no len()rtim=0&#13;&#10;Client Session Time: 16.43&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0x96d01090&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 13,86,0,5599&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19043 GPU: AMD Custom GPU 0405 (RADV VANGOGH) aticfx64.dll 32767.65535.65535.65535, GUID: 00000000-0004-0000-0000-000000000000&#13; &#10;</desyncdata></report>
</root>